woocommerce/woocommerce
# Update my-account-orders Perf Test to ensure the variable boundary operates with new codebase

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





An update to [fix: label improvement on my order pages](https://github.com/woocommerce/woocommerce/pull/48374#top) impacted one of the performance tests by changing the boundary required for the `my_account_order_id` variable. This small update ensures the test will run without warnings

Closes https://github.com/woocommerce/woocommerce-quality/issues/846 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Execute Perf test with `9.0.2`
2. Execute Perf test with `9.1.0 Beta`
3. Ensure both tests execute successfully and neither produce errors related to `invalid control character`



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
This is an update to a performance test id boundary and does not require a changelog

</details>
